Ginger/Ginger/AutomatePageLib/AddActionMenu/ActionsFactory.cs

```diff
@@ -181,6 +181,10 @@ static Act GenerateSelectedAction(Act selectedAction, Context mContext)
                             selectedAction.Platform = mContext.Platform;
                         }
                         instance = ((IObsoleteAction)selectedAction).GetNewAction();
+                        if(instance == null)
+                        {
```

Comment:
Add check if new Action exist for the Obsolete Action and if not then show new message which not recommending to use the Action and asking if still to add it Yes/No